pkg/api/controllers/target/list.go

```diff
@@ -30,5 +31,40 @@ func ListTargets(ctx *gin.Context) {
 		return
 	}
 
+	for _, target := range targets {
+		p, err := server.ProviderManager.GetProvider(target.ProviderInfo.Name)
```

Comment:
These error abortions are also a bit problematic.

If a single provider/manifest fails (e.g. the user uninstalls a provider), the user won't be able to list targets.

I suggest that, if an error occurs, we set `target.Options = err.String()` . This way, the user will be aware of the underlying issue and if it occurs, won't be able to view options.

Reply:
![Screenshot 2024-10-24 at 12 19 17](https://github.com/user-attachments/assets/9d404dd8-83c6-4437-a6e7-d6ac066a46ce)

Let's just prefix it with `Error: ` so it's more clear.

Reply:
![Screenshot 2024-10-24 at 12 30 04](https://github.com/user-attachments/assets/4f9a7b60-d533-4085-8992-8977bcc1cd7e)